nextcord/channel.py

```diff
@@ -1411,6 +1411,52 @@ def archived_threads(
             self.id, self.guild, limit=limit, joined=joined, private=private, before=before
         )
 
+    async def create_webhook(
+        self,
+        *,
+        name: str,
+        avatar: Optional[Union[bytes, Asset, Attachment, File]] = None,
+        reason: Optional[str] = None,
+    ) -> Webhook:
+        """|coro|
+
+        Requires :attr:`~.Permissions.manage_webhooks` permissions.
```

Comment:
```suggestion
        Creates a webhook for this channel.

        Requires :attr:`~.Permissions.manage_webhooks` permissions.
```
The description got removed.